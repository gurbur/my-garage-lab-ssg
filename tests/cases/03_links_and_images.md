## 링크 종합 테스트

이 문단에는 여러 종류의 링크가 있습니다.

표준 마크다운 링크는 이렇게 생겼습니다: [Google](https://google.com).
Obsidian 내부 글 링크는 이렇게 생겼습니다: [[(1) 첫 글]].
마지막으로, Obsidian 이미지 링크는 이렇게 생겼습니다:

![[image_preview.png]]

링크들 사이에도 일반 텍스트가 올 수 있습니다.
