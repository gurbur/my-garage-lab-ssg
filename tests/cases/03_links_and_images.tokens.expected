[HASH]
[HASH]
[TEXT]{" 링크 종합 테스트"}
[NEWLINE]
[NEWLINE]
[TEXT]{"이 문단에는 여러 종류의 링크가 있습니다"}
[DOT]
[NEWLINE]
[NEWLINE]
[TEXT]{"표준 마크다운 링크는 이렇게 생겼습니다: "}
[LBRACKET]
[TEXT]{"Google"}
[RBRACKET]
[LPAREN]
[TEXT]{"https://google"}
[DOT]
[TEXT]{"com"}
[RPAREN]
[DOT]
[NEWLINE]
[TEXT]{"Obsidian 내부 글 링크는 이렇게 생겼습니다: "}
[LBRACKET]
[LBRACKET]
[LPAREN]
[NUMBER]{"1"}
[RPAREN]
[TEXT]{" 첫 글"}
[RBRACKET]
[RBRACKET]
[DOT]
[NEWLINE]
[TEXT]{"마지막으로, Obsidian 이미지 링크는 이렇게 생겼습니다:"}
[NEWLINE]
[NEWLINE]
[EXCLAMATION]
[LBRACKET]
[LBRACKET]
[TEXT]{"image_preview"}
[DOT]
[TEXT]{"png"}
[RBRACKET]
[RBRACKET]
[NEWLINE]
[NEWLINE]
[TEXT]{"링크들 사이에도 일반 텍스트가 올 수 있습니다"}
[DOT]
[NEWLINE]
[EOF]
